src/ape/api/accounts.py

```diff
@@ -54,18 +56,21 @@ def alias(self) -> Optional[str]:
         return None
 
     @abstractmethod
-    def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
+    def sign_message(self, msg: Any, **signer_options) -> Optional[MessageSignature]:
```

Comment:
Are the changed related to signing messages supposed to be on this PR or are those part of https://github.com/ApeWorX/ape/pull/1614?